### DISCUSSION TITLE
Basic new user issues

### DISCUSSION THREAD

**Discussion by jrb-github** [vgalin/html2image#86] [Q&A] [answered]
First let me admit I am somewhat of a python n00b, but I've done my share of programming and using other python modules obtained via pip, and I feel like I'm missing something here.

I'm using python 3.7.9 on Windows 10.  I believe I'm using a venv, as recommended by people I work with who know more about python than me.

I performed a `pip install html2image` with no issues.

If I launch a console by just running `python` from a command prompt, typing `import html2image` yields no output, suggesting that it worked.  However, all examples I've seen suggest performing `from html2image import Html2image`.  This yields the following output for me:

![image](https://user-images.githubusercontent.com/49240289/202020111-6f137c49-d01a-442c-bd5d-1bf6674d2fe9.png)

So in trying the instructions in the README.md here, and various things similar to them, I haven't found syntax that works in my environment.  Does anyone have any suggestions on next steps?

**Comment by vgalin** [comment]
Hello, does performing the install using `python -m pip install html2image`
solves the problem?

Le mar. 15 nov. 2022 à 21:39, jrb-github ***@***.***> a
écrit :

**Reply by jrb-github** [reply]
Thanks, but unfortunately no.  I tried that, first by itself, then after a `pip uninstall html2image`, and in both cases this had no effect on the output of `from html2image import Html2image`.  :(

**Reply by vgalin** [reply, marked as answer]
@jrb-github my bad, the package is properly installed. I'm pretty sure I've spotted the issue, the "i" of `Html2image` needs to be capitalized (`Html2_I_mage` instead of `Html2_i_mage`), like so:
```
from html2image import Html2Image
```

**Reply by jrb-github** [reply]
Well that's embarrassing.  Not sure how I missed that, I should have tried copy-pasting before posting.  Sorry about that, thanks.

**Reply by vgalin** [reply]
Don't worry, it happens. Have a nice day!
